tools/runners/run-proxy.js

```diff
@@ -202,6 +203,20 @@ Object.assign(Proxy.prototype, {
       }
     }
 
+    const requestOptions = (() => {
```

Comment:
Suggested simpler/more readable way of doing the same:
```
    const requestOptions = {};

    if (self.proxyToSocket) {
      requestOptions.target = { socketPath: self.proxyToSocket };
    } else {
      requestOptions.target = 'http://' + self.proxyToHost + ':' + self.proxyToPort;
    }
```
No need for the awkward immediately invoked function pattern here.